Gemfile.lock

```diff
@@ -47,6 +47,7 @@ GEM
     parallel (1.17.0)
     parser (2.6.3.0)
       ast (~> 2.4.0)
+    private_address_check (0.4.1)
```

Comment:
Name: private_address_check
Version: 0.4.0
Advisory: CVE-2017-0909
Criticality: High
URL: https://github.com/jtdowney/private_address_check/pull/3
Title: private_address_check Ruby Gem Blacklist Bypass privilege escalation
Solution: Upgrade to >= 0.4.1.